source/docs/api-reference/simulation/simulation-intro.rst

```diff
@@ -73,6 +73,8 @@ Inputs and Outputs
 
 All ``SimState`` objects contain multiple inputs to manipulate the state of the device based on simulation physics calculations. For example, all device ``SimState`` objects have a supply voltage input:
 
+.. important:: Supply voltage is required to be configured on non-FRC projects.
```

Comment:
Supply voltage isn't a config, so "configured" isn't quite the right word here.
I also don't really like documenting this as "required in non-FRC" because it implies that FRC users don't have to worry about it at all (like with feedEnable), which isn't the case.